kivy/core/window/window_egl_rpi.py

```diff
@@ -44,7 +47,10 @@ def _create_window(self, w, h):
         display = egl.bcm_display_open(self._rpi_dispmanx_id)
         update = egl.bcm_update_start(0)
         element = egl.bcm_element_add(
-            update, display, self._rpi_dispmanx_layer, dst, src)
+            update, display, self._rpi_dispmanx_layer, dst, src,
+            self._rpi_dispmanx_overlay)
+        if self._rpi_dispmanx_overlay:
+            self.clearcolor = (0., 0., 0., 0.)
```

Comment:
I don't think we should do this by default as this goes against the docs https://github.com/kivy/kivy/blob/master/kivy/core/window/__init__.py#L480. Although this flag is new, it's still better to have the user set the desired opacity and update based on that.

Specifically, can you add `clearcolor` to the properties watched [here](https://github.com/kivy/kivy/blob/master/kivy/core/window/__init__.py#L994). Then, you won't need the `KIVY_BCM_DISPMANX_OVERLAY` variable, but instead, if the clear color opacity value is `>=1`, use `opaque` window mode, otherwise, use the `blend` mode and set `alpha.opacity` to the appropriate value based on `clearcolor[3]`.

If this can work, I think what I described would be preferable.